src/main/java/org/apache/cassandra/distributed/api/ICluster.java

```diff
@@ -89,7 +91,8 @@ static void setupLogging()
         {
             File root = Files.createTempDirectory("in-jvm-dtest").toFile();
             root.deleteOnExit();
-            String testConfPath = "test/conf/logback-dtest.xml";
+            String logConfigPropertyName = System.getProperty(PROPERTY_PREFIX + ".logConfigProperty", "logback.configurationFile");
+            String testConfPath = System.getProperty(PROPERTY_PREFIX + ".logConfigPath", "test/conf/logback-dtest.xml");
             Path logConfPath = Paths.get(root.getPath(), "/logback-dtest.xml");
```

Comment:
The copy still assumes `logback-dtest.xml`.  I wonder if we can just remove this since the path points to the original path rather than the copied version.

Reply:
agreed - fixed up and rebased on top of #7 (so the commits for this now looks weird, but didn't want to point it at your personal repo).